zzk/service/hoststate.go

```diff
@@ -332,7 +340,12 @@ func (l *HostStateListener) setInstanceState(containerExit <-chan time.Time, ssd
 			}
 
 			logger.Debug("Resumed paused container")
-		}
+		} else {
+			if cstate.Status != "started" && cstate.Status != "starting" {
+                                l.cleanUpContainers([]string{stateID}, true)
```

Comment:
The behaviour described in the if block manages already handles this information